### PR TITLE
fix: updated README

### PR DESCRIPTION
**Pull request by djfm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,104 @@
...
-
-### Content Creator 
+## API used by YouChoose
 
-POST /api/v3/creator/updateVideo
-GET /api/v3/creator/ogp/:url
-GET /api/v3/creator/videos/:publicKey
-GET /api/v3/creator/recommendations/:publicKey
+The API endpoints used by the YouChoose extension are automatically described in a dedicated page,
+including a swagger user interface to test them, under the "Sessings" tab within the extension.
```
Comment (ascariandrea) [inline README.md:104]: Settings*

File: README.md [hunk 2]
```diff
@@ -1,134 +1,104 @@
...
-Hash: cc4a9623ff47580332a5
-Version: webpack 2.1.0-beta.17
-Time: 8278ms
-        Asset     Size  Chunks             Chunk Names
-     popup.js  11.7 MB       0  [emitted]  popup
- dashboard.js  9.39 MB       1  [emitted]  dashboard
-       app.js     4 MB       2  [emitted]  app
-background.js  2.48 MB       3  [emitted]  background
-    + 825 hidden modules
+➡ npm run build:dev
```
Comment (ascariandrea) [inline README.md:31]: maybe it's not relevant to show the build "output"

File: README.md [hunk 3]
```diff
@@ -1,134 +1,104 @@
 # YouChoose.AI
 
-There are three components that compose this project.
+There are two components that compose this project.
 
 1. The browser extension. the core product of YouChoose.AI, sponsored by Ledger project round 3
 2. The backend of https://youtube.tracking.exposed, free software developed in 2018 under the ERC project ALEX (https://algorithms.exposed). This package since 2019 is maintained as an independent mechanism for Youtube algorithm accountability and enhanced to be the backend supporting YouChoose.
```
Comment (ascariandrea) [inline README.md:6]: We can add a link to the `yttrex` repo here

**Review by ascariandrea — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (djfm) [reply README.md:104]: lol, thx

File: README.md (hunk 2 above)
Comment (djfm) [reply README.md:31]: indeed

File: README.md (hunk 3 above)
Comment (djfm) [reply README.md:6]: yes

File: README.md [hunk 4]
```diff
@@ -1,134 +1,70 @@
...
-       app.js     4 MB       2  [emitted]  app
-background.js  2.48 MB       3  [emitted]  background
-    + 825 hidden modules
+npm run watch
 ```
 
-At this point you should load it in your browser. Firefox and Chrome have slightly different action required.
+At this point you can load it in your browser. Firefox and Chrome have slightly different action required.
+
+To create a pruction build, run:
```
[the remark this answers is not in the document]
Comment (djfm) [reply README.md:30]: wow I don't know what's happening to me today...

**Review by ascariandrea — APPROVED** [review]

**Pull request merged**